sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -351,15 +351,27 @@ def on_response(self, request, response):
         except Exception as err:  # pylint: disable=broad-except
             _LOGGER.debug("Failed to log response: %s", repr(err))
 
+class _HiddenClassProperties(type):
+    # Backward compatible for DEFAULT_HEADERS_WHITELIST but no documentation needed
+    # https://github.com/Azure/azure-sdk-for-python/issues/26331
 
-class HttpLoggingPolicy(SansIOHTTPPolicy):
+    @property
+    def DEFAULT_HEADERS_WHITELIST(cls):
+        return cls.DEFAULT_HEADERS_ALLOWLIST
```

Comment:
Do we want to issue a deprecation warning if this is used? Or will that be too noisy for the time being?

Reply:
I think it's too noisy, we don't have any reasons to remove it for now. Maybe if we feel it becomes a maintenance burden, we can add it later when it was not documented for months